scapy/contrib/avs.py

```diff
@@ -21,6 +21,8 @@
 from scapy.fields import *
 from scapy.layers.dot11 import *
 
+LINKTYPE_IEEE802_11_AVS = 163
```

Comment:
Please create a constant value `DLT_IEEE802_11_RADIO_AVS = 163` in `data.py` and use it instead.